src/jimgw/prior.py

```diff
@@ -275,6 +275,42 @@ def __init__(
         )
 
 
+@jaxtyped(typechecker=typechecker)
+class GaussianPrior(SequentialTransformPrior):
+    mu: float = 0.0
+    sigma: float = 1.0
```

Comment:
I tried changing the typing hint accrodingly in my codespace but it doesn't seem to allow me to push to your fork. Let me know if you see any changes on this